examples/usecases/aci_mcp/env.example

```diff
@@ -0,0 +1,25 @@
+# CAMEL-AI and ACI Environment Configuration Template
+
+# How to use:
+# 1. Make a copy of this file and name it .env (e.g., `cp .env.template .env`)
+# 2. Fill in the values for each variable below.
+# 3. Ensure the .env file is in the same directory as your main scripts (e.g., aci-portia/)
+#    or that your scripts load it correctly if it's located elsewhere.
+#
+# Add .env to your .gitignore file.
+
+# --- Google Generative AI Configuration ---
+# Required for CAMEL-AI to use Google's language models (e.g., Gemini).
+# Paste your gemini api key here and configure the model in the config.py
+GOOGLE_API_KEY="YOUR_GOOGLE_API_KEY_HERE"
```

Comment:
Let's name it as `GEMINI_API_KEY`. `GOOGLE_API_KEY` usually is for google search